lib/fizzy/instructions.hpp

```diff
@@ -39,4 +41,13 @@ struct InstructionMetrics
 
 const InstructionMetrics* get_instruction_metrics_table() noexcept;
 
+
+struct InstructionType
```

Comment:
Can you add a comment here that wasm 1.0 spec only has instructions which take at most 2 items and return at most 1 item?